pkgs/dartpad_ui/lib/main.dart

```diff
@@ -653,7 +665,7 @@ class DartPadAppBar extends StatelessWidget implements PreferredSizeWidget {
           bottom: bottom,
           actions: [
             // Hide the Install SDK button when the screen width is too small.
-            if (constraints.maxWidth > smallScreenWidth)
+            if (constraints.maxWidth >= minLargeScreenWidth)
```

Comment:
Are these changes intentional? What's the reason behind changing the breakpoint behavior?

Reply:
I want to set width to this value and I want screen to be still wide, to test for overload.
This was my idea, that did not work out, because test rendering sizes are little different than on mac and web, may be because they are on ubuntu.
But it is still good, because eventually we may want to test for overload for web platform.

Reply:
How does this affect the user experience?

Reply:
Sorry, not 'overload', but 'overflow'.

Overflow means there may be visual cuts that developer did not expect. In debug mode it is error, in prod it is not.

See: https://medium.com/@ugamakelechi501/solving-widget-overflow-issues-in-flutter-layouts-a-beginner-s-guide-d56b5de5af2d

Reply:
Change of > to >= will not change user experience.